keyboards/handwired/slash/readme.md

```diff
@@ -0,0 +1,19 @@
+# slash
+
+![slash](https://i.imgur.com/MYEfSE0.jpg)
+
+I found this project a while ago on [Reddit](https://www.reddit.com/r/MechanicalKeyboards/comments/6v3b6f/build_slash_keyboard_handwired_bluetooth/). This keyboard uses 61 keys in a 8x8 matrix. That bring the required pins to 16 which leaves 1 pin free. Unfortunately I couldn't get the firmware to compile on kbfirmware.com(the way [ND1BV](https://imgur.com/user/ND1BV) described it). So I decided to create the config myself and share it. I changed the wiring of the pins a little bit showed [here](https://i.imgur.com/WZ93xHx.png)
+
+Keyboard Maintainer: [asdftemp](https://github.com/asdftemp)
+Hardware Supported: Adafruit Feather 32u4 Bluefruit LE
```

Comment:
Lines 7 and 8 here should both end with two spaces, so GitHub line breaks this properly, instead of doing this:

![image](https://user-images.githubusercontent.com/18669334/67728399-535ef780-f9aa-11e9-8b75-44779c9ed618.png)

```Markdown
Keyboard Maintainer: [asdftemp](https://github.com/asdftemp)  
Hardware Supported: Adafruit Feather 32u4 Bluefruit LE  
```

Reply:
Ah tought the double space should be removed. I have added them to to the three items on the list